### ISSUE TITLE
[feat] Get bundle tools from local path

### ISSUE THREAD

**Issue by taoqf** [tauri-apps/tauri#11801]
### Describe the problem

I cannot build app because I could not access to network.

### Describe the solution you'd like

1. Release these resources with stable tauri.
2. Support an option like `--localpath` to copy local files from.
3. Both above.

### Alternatives considered

1. Mirror, I searched and find env `TAURI_BUNDLER_TOOLS_GITHUB_MIRROR` and `TAURI_BUNDLER_TOOLS_GITHUB_MIRROR_TEMPLATE`, and I prefer local file, it's more stable.

**Comment by FabianLars** [comment]
> Release these resources with stable tauri.

Some of the tools used are too large to be included in tauri itself imo

> Support an option like --localpath to copy local files from.

I agree that an explicit approach makes sense, but for now simply adding the files to the `~/.cache/tauri` (linux) or `AppData/Local/tauri` (windows) directory yourself _should_ skip the downloads.

**Comment by taoqf** [comment]
I checked `fn prepare_tools`, Thanks.
Should I close this issue?

**Comment by FabianLars** [comment]
No, i think we should think about some proper support for this, even if it's just documentation on what tools are needed where.

**Comment by taoqf** [comment]
@FabianLars [local-tools](https://github.com/tauri-apps/tauri/blob/4c3f04773534c0b33ea81bcff6dd7719ef265662/crates/tauri-bundler/src/bundle/linux/appimage.rs#L43) is '.tauri', but the docs said [.tauri-tools](https://github.com/tauri-apps/tauri/blob/4c3f04773534c0b33ea81bcff6dd7719ef265662/crates/tauri-cli/config.schema.json#L1926).

**Comment by FabianLars** [comment]
thanks https://github.com/tauri-apps/tauri/pull/12409
